doc/source/how_to_use.rst

```diff
@@ -54,6 +54,28 @@ For the serious application developer, all of the functionality in pyqtgraph is
 See the designer documentation for more information on promoting widgets. The "VideoSpeedTest" and "ScatterPlotSpeedTest" examples both demonstrate the use of .ui files that are compiled to .py modules using pyuic5 or pyside-uic. The "designerExample" example demonstrates dynamically generating python classes from .ui files (no pyuic5 / pyside-uic needed).
 
 
+HiDPI Displays
+--------------
+
+PyQtGraph has a method :func:`mkQApp <pyqtgraph.Qt.mkQApp>` that by default sets what we have tested to be the best combination of options to support hidpi displays, when in combination with non-hidpi secondary displays.  For your application, you may have instantiated ``QApplication`` yourself, in which case we advise setting these options *before* runing ``QApplication.exec_()``.  
+
+For Qt6 bindings, this functionally "just works" without having to set any attributes.
+
+On Versions of Qt >= 5.14 and < 6; you can get ideal behavior with the following lines::
+
+    os.environ["QT_ENABLE_HIDPI_SCALING"] = "1" 
```

Comment:
typo: variable name should be "QT_ENABLE_HIGHDPI_SCALING"

Reply:
o no.... this sort of calls into question all my testing 🤦  I'll submit a PR to fix it tomorrow.